02week/blog/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>My Blog</title>
+  <link rel="stylesheet" href="css/style.css" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
+  <header>
+    <nav>
+      <a href="mailto:demarcospears22@gmail.com" target="_blank"><i class="fas fa-at"></i></a>
+      <a href="https://www.linkedin.com/in/demarco-spears22" target="_blank"><i class="fab fa-linkedin"></i></a>
+      <a href="https://www.instagram.com/dspears_2/" target="_blank"><i class="fab fa-instagram"></i></a>
+      <a href="file:///C:/Users/DeMarco/Desktop/web-workbook/02week/blog/index.html" target="_blank"> <i class="fab fa-blogger"></i></a>
+      <a href="file:///C:/Users/DeMarco/Desktop/web-workbook/01week/resume/index.html" target="_blank"><i class="far fa-file"></i></a>
+      <a href="file:///C:/Users/DeMarco/Desktop/web-workbook/02week/bio/index.html" target="_blank"><i class="fas fa-user-alt"></i></a>
```

Comment:
Let's revisit these links.. you don't need to put the entire pathway, only the path that is relative to your current file.

I.e. it would probably look more similar to

`href="../02week/bio/index.html"`